snarkos/ledger/mod.rs

```diff
@@ -95,36 +95,11 @@ impl<N: Network> Ledger<N> {
         self.ledger.write().add_to_memory_pool(transaction)
     }
 
-    /// Advances the ledger to the next block.
-    pub async fn advance_to_next_block(self: &Arc<Self>) -> Result<Block<N>> {
```

Comment:
Unremove this

Reply:
it's unused, though; mark it as `#[allow(dead_code)]`?

Reply:
"add_to_memory_pool" is also technically unused by the client then.

We will need to reimplement these later so it is easier to have them here and refactor as oppose to removing them.